src/components/WeatherTile/useWeatherCode.jsx

```diff
@@ -0,0 +1,76 @@
+import { useEffect, useState } from "react";
+import SunnySVG from "../../assets/iconsWeather/weather-sun.svg?react";
+import CloudySVG from "../../assets/iconsWeather/weather-cloudy.svg?react";
+import FogSVG from "../../assets/iconsWeather/weather-fog.svg?react";
+import RainSVG from "../../assets/iconsWeather/weather-rain.svg?react";
+import RainCloudySVG from "../../assets/iconsWeather/weather-sunny-cloudy-rainy.svg?react";
+import SnowSVG from "../../assets/iconsWeather/weather-snow.svg?react";
+import ThunderstormSVG from "../../assets/iconsWeather/weather-thunderstorm.svg?react";
+
+const useWeatherCode = (weatherCode) => {
+  const ICON_CLASSES = "mt-6 fill-white w-10";
+
+  const [icon, setIcon] = useState(null);
+
+  useEffect(() => {
+    let weatherCondition = null;
+    switch (weatherCode) {
+      case 0:
+        weatherCondition = <SunnySVG className={ICON_CLASSES} />;
```

Comment:
SVG elements should have an alt tag just like Image elements describing what they are